.travis-ocaml.sh

```diff
@@ -43,42 +43,12 @@ UBUNTU_TRUSTY=${UBUNTU_TRUSTY:-"0"}
 # Install XQuartz on OSX
 INSTALL_XQUARTZ=${INSTALL_XQUARTZ:-"true"}
 
-install_on_linux () {
-  case "$OCAML_VERSION,$OPAM_VERSION" in
-    3.12,1.2.2)
-       OCAML_VERSION=4.02; OCAML_FULL_VERSION=3.12.1
-       ppa=avsm/ocaml42+opam12 ;;
-    4.00,1.2.2)
-       OCAML_VERSION=4.02; OCAML_FULL_VERSION=4.00.1
-       ppa=avsm/ocaml42+opam12 ;;
-    4.01,1.2.2)
-       OCAML_VERSION=4.02; OCAML_FULL_VERSION=4.01.0
-       ppa=avsm/ocaml42+opam12 ;;
-    4.02,1.1.2) OCAML_FULL_VERSION=4.02.3; OPAM_SWITCH=${OPAM_SWITCH:-system}; ppa=avsm/ocaml42+opam11 ;;
-    4.02,1.2.0) OCAML_FULL_VERSION=4.02.3; OPAM_SWITCH=${OPAM_SWITCH:-system}; ppa=avsm/ocaml42+opam120 ;;
-    4.02,1.2.1) OCAML_FULL_VERSION=4.02.3; OPAM_SWITCH=${OPAM_SWITCH:-system}; ppa=avsm/ocaml42+opam121 ;;
-    4.02,1.2.2) OCAML_FULL_VERSION=4.02.3; OPAM_SWITCH=${OPAM_SWITCH:-system}; ppa=avsm/ocaml42+opam12 ;;
-    4.03,1.2.2)
-       OCAML_VERSION=4.02; OCAML_FULL_VERSION=4.03.0
-       ppa=avsm/ocaml42+opam12 ;;
-    4.04,1.2.2)
-        OCAML_VERSION=4.02; OCAML_FULL_VERSION=4.04.2
-        ppa=avsm/ocaml42+opam12 ;;
-    4.05,1.2.2)
-        OCAML_VERSION=4.02; OCAML_FULL_VERSION=4.05.0
-        ppa=avsm/ocaml42+opam12 ;;
-    4.06,1.2.2)
-        OCAML_VERSION=4.02; OCAML_FULL_VERSION=4.06.1
-        ppa=avsm/ocaml42+opam12 ;;
-    4.07,1.2.2)
-        OCAML_VERSION=4.02; OCAML_FULL_VERSION=4.07.0
-        ppa=avsm/ocaml42+opam12 ;;
-    *) echo "Unknown OCAML_VERSION=$OCAML_VERSION OPAM_VERSION=$OPAM_VERSION"
-       echo "(An unset OCAML_VERSION used to default to \"latest\", but you must now specify it."
-       echo "Try something like \"OCAML_VERSION=3.12\", \"OCAML_VERSION=4.07\", or see README-travis.md at https://github.com/ocaml/ocaml-ci-scripts )"
-       exit 1 ;;
-  esac
+install_opam2 () {
+  sudo wget https://github.com/ocaml/opam/releases/download/2.0.0-rc3/opam-2.0.0-rc3-x86_64-linux -O /usr/local/bin/opam ;;
```

Comment:
mmh maybe a `sudo chmod +x /usr/local/bin/opam` is going to be missing at some point I guess.